### PR TITLE
chore: address various linter issues

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Addressing linter issues, mostly from explorer

# To Test

1. Explorer should work as before

### CODE REVIEW

File: apps/explorer/src/Console.tsx
Comment (alfetopito) [inline apps/explorer/src/Console.tsx:1]: Was used for debugging mobile devices

**Review by shoom3301 — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @alfetopito , great job!
1 question: is this OK that the view of the 'Show more' button is changed here? 
image
image

**Comment by alfetopito** [comment]
> Hey @alfetopito , great job! 1 question: is this OK that the view of the 'Show more' button is changed here? image image

Aha!
I suspected something would break, but did not see it when testing.
Will fix it.

**Comment by elena-zh** [comment]
Hey @alfetopito , not fixed on the AppData - Decode page :) 
image

**Comment by alfetopito** [comment]
> Hey @alfetopito , not fixed on the AppData - Decode page :) image

Now it's fixed 🤞 

image

**Review by shoom3301 — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
Looks good now!

**Pull request merged**